utils/src/macros.rs

```diff
@@ -0,0 +1,33 @@
+/// Print package related information.
+/// This includes the package name and version. The git hash will also be included if supplied.
+#[macro_export]
+macro_rules! print_package_info {
```

Comment:
Should this make use of the package_info macro?

Reply:
I chose not to do this because that is making use of heap allocations via boxing and I thought it would be better to avoid that.